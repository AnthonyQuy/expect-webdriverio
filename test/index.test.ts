import { test, expect } from 'vitest'
import { setOptions, expect as expectExport, matchers } from '../src/index.js'

test('index', () => {
    expect(setOptions.name).toBe('setDefaultOptions')
    expect(expectExport).toBeDefined()
    expect(matchers).toBeDefined()
})
