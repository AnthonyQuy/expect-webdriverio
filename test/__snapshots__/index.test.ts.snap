// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test/index.test.ts > supports snaptshot testing 1`] = `"foobar"`;
